### PR TITLE
3.7.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 3.7.0 (2020-05-01)
+
+- Updated the visual styles of the column resize indicator. [#116](https://github.com/blackbaud/skyux-grids/pull/116)
+- Removed dependency on `moment`. [#116](https://github.com/blackbaud/skyux-grids/pull/116)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md:4]: The PR link is the same as visual styles entry. Is that correct?

Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md:4]: That is correct. We did them both in the same.

Comment (blackbaud-johnly) [reply CHANGELOG.md:4]: We should have "a" before "dependency.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-3.7.0-release](https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-3.7.0-release)

**Pull request merged**